regressionTests.cmake

```diff
@@ -99,37 +100,39 @@ add_test_compareECLFiles(CASENAME spe1_precsalt
                          REL_TOL ${rel_tol}
                          DIR spe1_precsalt)
 
+set(network_test_args --enable-tuning=true --time-step-control=newtoniterationcount --time-step-control-target-newton-iterations=15 --time-step-control-growth-rate=3.0 --relaxed-max-pv-fraction=0.0 --tolerance-cnv=1e-3 --max-newton-iterations-with-inner-well-iterations=15)
```

Comment:
Tiny nit: Would you mind splitting this across more lines for those of us who don't have very wide terminals, please?  Preferably one setting per line if possible, e.g.,
```CMake
set (network_test_args
    --enable-tuning=true
    --time-step-control=newtoniterationcount
    # ...
    --max-newton-iterations-with-inner-well-iterations=15
  )
```

Reply:
I was unsure if the line breaks would cause commands to be split and fail, but if it works as you suggest I agree it is better.

Reply:
> I was unsure if the line breaks would cause commands to be split and fail, but if it works as you suggest I agree it is better.

Both `set()` commands do the same thing.  The easiest way to see this is to run the script below (save commands to a file `a.cmake` and run the command `cmake -P a.cmake` from the shell) and to notice that both `a` and  `a2` expand to the same list.

---

```cmake
set (a
  b
  c
  d
  e)

set (a2 b c d e)

message (STATUS "a  = " "${a}")
message (STATUS "a2 = " "${a2}")
```